src/field_weather.c

```diff
@@ -1104,3 +1104,11 @@ void ResetPreservedPalettesInWeather(void)
 {
     sPaletteColorMapTypes = sBasePaletteColorMapTypes;
 }
+
+bool8 IsWeatherAlphaBlend(void)
+{
+    return gWeatherPtr->currWeather == WEATHER_FOG_HORIZONTAL || 
+           gWeatherPtr->currWeather == WEATHER_FOG_DIAGONAL   ||
+           gWeatherPtr->currWeather == WEATHER_UNDERWATER_BUBBLES ||
+           gWeatherPtr->currWeather == WEATHER_UNDERWATER;
```

Comment:
I believe we surround these kinds of returns with parentheses in the rest of the repo